c2cgeoform/scaffolds/c2cgeoform/+package+/models/c2cgeoform_demo.py

```diff
@@ -24,11 +24,9 @@
 from c2cgeoform.ext import colander_ext, deform_ext
 from c2cgeoform.models import FileData
 
+from .. import _
```

Comment:
wonder whether it make sense being be more explicit.

Reply:
like `from {{package}} import _`?
+1 for me :-)

Reply:
Yes, I would prefer ``from {{package}} import _`` but this is not a template.

Reply:
Then rename it ...